### PR TITLE
[v2.7] K8s july patches

### PR DESCRIPTION
**Pull request by vardhaman22**
for issue: https://github.com/rancher/rancher/issues/42009

### CODE REVIEW

File: pkg/rke/k8s_rke_system_images.go [hunk 1]
```diff
@@ -8902,6 +8902,50 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/mirrored-pause:3.7",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.21.1",
 		},
+		// Enabled in Rancher v2.7.6
```
Comment (snasovich) [inline pkg/rke/k8s_rke_system_images.go]: So far we're heavily leaning towards releasing OOB _before_ 2.7.6

Same for other introduced versions.

```suggestion
		// Enabled out of band post v2.7.5
```

**Review by snasovich — DISMISSED** [review]
LGTM, confirmed the only thing that changed is the versions on `Kubernetes` image.
One nit regarding comment which version it will be enabled in.
Feel free to consider my approval "transferrable" if addressing that one nit is the only difference in future update to this PR.

**Review by jiaqiluo — DISMISSED** [review]
Please address Sergey's comment. Otherwise LGTM

File: pkg/rke/k8s_rke_system_images.go (hunk 1 above)
Comment (vardhaman22) [reply pkg/rke/k8s_rke_system_images.go]: done

**Review by snasovich — DISMISSED** [review]

**Review by jiaqiluo — DISMISSED** [review]

**Review by snasovich — APPROVED** [review]

**Review by kinarashah — APPROVED** [review]

**Review by jiaqiluo — APPROVED** [review]

**Pull request merged**